### ISSUE TITLE
Make clear you need to have a running PostgreSQL instance

### ISSUE THREAD

**Issue by giuscri** [gabro/rps-to-the-max#1]
I think this should be stated in a README: as I understand this, to run this code an instance of PostgreSQL must be running, `rps` database and `rps` user with superuser power must be created.
```
$ pg_ctl -D /usr/local/var/postgres start
$ psql postgres
postgres=# create user rps with superuser;
postgres=# create database rps;
```

**Comment by gabro** [comment]
Yep, thanks. This was intended as an experiment and I've used this for an internal presentation. Anyway, I'll gladly merge a PR updating the readme, if you have time :)
